content/enterprise_influxdb/v1.9/about-the-project/release-notes-changelog.md

```diff
@@ -9,6 +9,30 @@ menu:
     parent: About the project
 ---
 
+## 1.9.7 [2022-05-11]
+
+### Features
+- Add experimental passive node feature; expose passive node feature to influxd-ctl and the API.
+- Throttle inter-node data replication, both incoming writes and hinted hand-off, when errors are encountered.
+
+#### Flux updates
+- Add [http requests package](/{{< latest "flux" >}}/stdlib/experimental/http/requests/).
+- Add [isType()](/{{< latest "flux" >}}/stdlib/types/istype/) function.
+- Add [display()](/{{< latest "flux" >}}/stdlib/universe/display/) function.
+- Enhancements to the following functions: [increase()](/{{< latest "flux" >}}/stdlib/universe/increase/), [sort()](/{{< latest "flux" >}}/stdlib/universe/sort/), [derivative()](/{{< latest "flux" >}}/stdlib/universe/derivative/), [union()](/{{< latest "flux" >}}/stdlib/universe/union/), [timeShift()](/{{< latest "flux" >}}/stdlib/universe/timeshift/), vectorization to applicable functions such as [map()](/{{< latest "flux" >}}/stdlib/universe/map/).
+- Add TCP connection pooling to [mqtt.publish()](/{{< latest "flux" >}}/stdlib/experimental/mqtt/publish/) function when called in a map() function.
+
```

Comment:
```suggestion
  - Improve influxd-ctl error messages when invalid JSON is received
  
```

Reply:
I can add this under the error messaging section, don't think this should go under the flux updates section